docs/node-operators/get-started/setup-validator.md

```diff
@@ -163,33 +243,51 @@ Then ensure the `max_packet_msg_payload_size` is at least 16384.
 
+
 ### Update Tendermint genesis
 To start successfully, tendermint needs the genesis file from the network you will be trying to join. This file need to be located in `YOUR_TENDERMINT_HOME/config/genesis.json`. Download the genesis file and use it to replace the genesis in your config.
 
 You can find genesis files: 
-* In the [networks repository](https://github.com/vegaprotocol/networks) for the mainnet network. 
-* In the [networks internal repository](https://github.com/vegaprotocol/networks-internal) for sandbox and other test networks. Note: For sandbox, the genesis must be a URL to a remote file, not saved locally on disk.
+* In the [networks repository ↗](https://github.com/vegaprotocol/networks) for the mainnet network. 
+* In the [networks internal repository ↗](https://github.com/vegaprotocol/networks-internal) for sandbox and other test networks. Note: For sandbox, the genesis must be a URL to a remote file, not saved locally on disk.
 
-For example, to join mainnet you will need the following [genesis file](https://github.com/vegaprotocol/networks/blob/master/mainnet1/genesis.json).
+For example, to join mainnet you will need the following [genesis file ↗](https://github.com/vegaprotocol/networks/blob/master/mainnet1/genesis.json).
 
 ## Synchronise your node
-You can either start the node from a snapshot (recommended, particularly for a long chain), or replay the full chain history to get up-to-date.
+You can either start the node from a snapshot (recommended, particularly for a long chain), or replay the full chain history to get up-to-date. If using Visor for protocol upgrades, follow the instructions below to start the node with Visor.
 
 ### Start node from a snapshot
 [Snapshots](../how-to/use-snapshots.md): Use a recent network snapshot to start your node without having to replay the entire chain.
 
-After you start from snapshot, start your node by running the following command:
-```vega start```
+After you choose the snapshot you're starting from: 
+
+**If you're using Visor**, start your node by running the service manager of your choice and use the following command:
+
+```shell
+visor run --home "VISOR_HOME_PATH"
+```
+
+**If you're not using Visor**, start your node by running the following command e.g. for sandbox:
+
+```shell
+vega start --home="YOUR_VEGA_HOME_PATH" --nodewallet-passphrase-file="YOUR_PASSPHRASE_FILE_PATH" --network=sandbox
+```
 
 ### Replay from genesis
 To replay all history from genesis: 
 
-You can set a genesis file when starting the node with the following command, e.g for sandbox:
+You can set a genesis file when starting the node with the following command, e.g. for sandbox:
 
 ```shell
 vega start --home="YOUR_VEGA_HOME_PATH" --nodewallet-passphrase-file="YOUR_PASSPHRASE_FILE_PATH" --network=sandbox
 ```
 
+If using Visor, start your node by running the service manager of your choice, including the required args (flags) for network, etc, and use the following command:
```

Comment:
"If using Visor, start your node by running the service manager of your choice" - > "If using Visor, start the Visor by running a service manager of your choice and do not forget to configure it."

Something like that makes more sense... Not the exact words.. :) Because the text above kind of give me feeling that I should start the Vega node with service manager and then run the Visor command.